src/components/Input/Input.jsx

```diff
@@ -66,6 +70,8 @@ Input.propTypes = {
   disabled: PropTypes.bool,
   /** It colors the field in red. */
   hasError: PropTypes.bool,
+  /** Aria-required value  */
```

Comment:
I'd suggest something like:
```suggestion
  /** Marks the input as required  */
```